src/bio.c

```diff
@@ -163,33 +185,23 @@ void bioCreateFsyncJob(int fd, long long offset) {
 
 void *bioProcessBackgroundJobs(void *arg) {
     bio_job *job;
-    unsigned long type = (unsigned long) arg;
+    unsigned long worker = (unsigned long) arg;
```

Comment:
to reduce the diff, maybe keep `type` (as a worker type), you'll still have `job->type` for the job type.

Reply:
I think overloading the word "type" here will be quite confusing. Is it really worth the slightly fewer lines in the diff?